README.md

```diff
@@ -26,6 +26,13 @@ Getting Started
 
 ### Automatic Install using `pip`
 
+According to the `manylinux1` policy described in
+[PEP 513](https://www.python.org/dev/peps/pep-0513/), binary packages
+**should depend only on an extremely limited set of external shared libraries.**
```

Comment:
Because this is the requirement originally by us, rewrite as below:
- Removing emphasis marker
- "packages should depend" -> "packages are required to depend"